src/main/javascript/src/views/assignment/TerracottaBuilder.vue

```diff
@@ -153,6 +156,9 @@ export default {
         (c) => parseInt(c.conditionId) === parseInt(this.condition_id)
       );
     },
+    canClearAll() {
+      return this.assessment.questions.length > 0 || (this.assessment.submissions.length === 0);
```

Comment:
Should the logic here be AND instead of OR?

Reply:
I don't believe so, if either of these is true the button for clearing questions should be disabled is the way I understood it.

Reply:
Okay, maybe I'm being dense, but it seems like canClearAll will return true when an assessment has one or more questions even if it also has submissions.

Reply:
I apologize, I was the one being dense! yes I definitely had this backward.